models/rtctrl_match_comm_regex_term.go

```diff
@@ -0,0 +1,88 @@
+package models
+
+import (
+	"fmt"
+	"strconv"
+
+	"github.com/ciscoecosystem/aci-go-client/container"
+)
+
+const (
+	DnrtctrlMatchCommRegexTerm        = "uni/tn-%s/subj-%s/commrxtrm-%s"
+	RnrtctrlMatchCommRegexTerm        = "commrxtrm-%s"
+	ParentDnrtctrlMatchCommRegexTerm  = "uni/tn-%s/subj-%s"
+	RtctrlmatchcommregextermClassName = "rtctrlMatchCommRegexTerm"
+)
+
+type MatchRuleBasedonCommunityRegularExpression struct {
+	BaseAttributes
+	MatchRuleBasedonCommunityRegularExpressionAttributes
+}
+
+type MatchRuleBasedonCommunityRegularExpressionAttributes struct {
+	Annotation string `json:",omitempty"`
+	CommType   string `json:",omitempty"`
+	Name       string `json:",omitempty"`
+	Regex      string `json:",omitempty"`
+}
+
+func NewMatchRuleBasedonCommunityRegularExpression(rtctrlMatchCommRegexTermRn, parentDn, description string, rtctrlMatchCommRegexTermAttr MatchRuleBasedonCommunityRegularExpressionAttributes) *MatchRuleBasedonCommunityRegularExpression {
+	dn := fmt.Sprintf("%s/%s", parentDn, rtctrlMatchCommRegexTermRn)
+	return &MatchRuleBasedonCommunityRegularExpression{
+		BaseAttributes: BaseAttributes{
+			DistinguishedName: dn,
+			Description:       description,
+			Status:            "created, modified",
+			ClassName:         RtctrlmatchcommregextermClassName,
+			Rn:                rtctrlMatchCommRegexTermRn,
+		},
+		MatchRuleBasedonCommunityRegularExpressionAttributes: rtctrlMatchCommRegexTermAttr,
```

Comment:
Object has nameAlias in the definition did you remove it?

Reply:
yes